code/src/App.js

```diff
@@ -1,9 +1,37 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, createStore} from '@reduxjs/toolkit'
+
+import { tasks } from './reducers/tasks'
+
+import { TaskList } from './components/TaskList'
+
+const reducer = combineReducers({
+  tasks: tasks.reducer
+})
+
+//NEW STORE WITH PERSISTED STATE
+const persistedStateJSON = localStorage.getItem("reduxState")
+let persistedState = {}
+
+if (persistedStateJSON) {
+  persistedState = JSON.parse(persistedStateJSON)
+}
+
+const store = createStore(
+  reducer, 
+  persistedState,
+  window.__REDUX_DEVTOOLS_EXTENSION__&& window.__REDUX_DEVTOOLS_EXTENSION__()
+)
+
+store.subscribe(() => {
+  localStorage.setItem("reduxState", JSON.stringify(store.getState()))
+})
```

Comment:
So nice feature to have this local storage! Want to implement this on my own code!